code/game/objects/items/storage/belt.dm

```diff
@@ -257,22 +244,21 @@
 		/obj/item/restraints/handcuffs,
 		/obj/item/assembly/flash/handheld,
 		/obj/item/clothing/glasses,
-		/obj/item/ammo_casing/shotgun,
+		/obj/item/ammo_casing,
 		/obj/item/ammo_box,
 		/obj/item/storage/box/rubbershot,
 		/obj/item/storage/box/lethalshot,
 		/obj/item/storage/box/breacherslug,
 		/obj/item/storage/box/beanbag,
 		/obj/item/reagent_containers/food/snacks/donut,
-		/obj/item/kitchen/knife/combat,
+		/obj/item/kitchen/knife,
 		/obj/item/flashlight/seclite,
-		/obj/item/melee/classic_baton/telescopic,
```

Comment:
Why is this removed?

Reply:
same reason as the mining belt's bottles removal; it's subtype supported.